website/content/docs/enterprise/supported-versions.mdx

```diff
@@ -20,15 +20,24 @@ A major release is identified by a change in the first (X) or second (Y) digit i
 As a best practice, HashiCorp expects customers to stay current within two (2) releases from the latest major release in order to receive optimal support.
 Release updates for Boundary Enterprise can be found at https://releases.hashicorp.com.
 
-## Control plane and worker availability
+## Control plane and worker compatability
 
-Although major releases are supported as outlined within the support periods section above, within a Boundary Enterprise deployment API backwards compatibility is only supported between the control plane and workers from the prior “major release”.
+Although major releases are supported as outlined within the support periods section above, within a Boundary Enterprise deployment API backwards compatibility is only supported between the control plane and workers from the prior “major release”. Using workers on newer versions than the control plane they are registered with is not supported.
```

Comment:
I'm not sure I understand what "registered with" means.  Perhaps: "Using a worker with version that is newer than the control plane they connect to is not supported."

Reply:
I like this as well, I think it's more accurate for multi-hop scenarios